scipy/integrate/_quadpack_py.py

```diff
@@ -663,6 +700,49 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
         handles integration over infinite intervals. The infinite range is
         mapped onto a finite interval and subsequently the same strategy as
         in ``QAGS`` is applied.
+    qagpe
+        serves the same purposes as QAGS, but also allows the
```

Comment:
I commented on this elsewhere, but "QAGS" is not defined. It should be defined if it is going to be used. Same with QAWO and any other acronyms.